File: player-lib/src/main/java/com/gtbluesky/mediamatrix/player/util/LogUtil.java
package com.gtbluesky.mediamatrix.player.util;

import android.util.Log;

public class LogUtil {

    private static final boolean SHOW_LOG = true;
    private static final String TAG = "gtbluesky";

    public static void v(String msg) {
        if (!SHOW_LOG) {
            return;
        }
        Log.v(TAG, msg);
    }

    public static void d(String msg) {
        if (!SHOW_LOG) {
            return;
        }
        Log.d(TAG, msg);
    }

    public static void i(String msg) {
        if (!SHOW_LOG) {
            return;
        }
        Log.i(TAG, msg);
    }

    public static void w(String msg) {
        if (!SHOW_LOG) {
            return;
        }
        Log.w(TAG, msg);
    }

    public static void e(String msg) {
        if (!SHOW_LOG) {
            return;
        }
        Log.e(TAG, msg);
    }
}

File: player-lib/src/main/java/com/gtbluesky/mediamatrix/player/util/LogUtil.kt
package com.gtbluesky.mediamatrix.player.util

import android.util.Log

object LogUtil {
    private const val SHOW_LOG = true
    private const val TAG = "LogUtil"

    fun v(msg: String?) {
        if (!SHOW_LOG) {
            return
        }
        if (msg.isNullOrEmpty()) {
            return
        }
        Log.v(TAG, msg)
    }

    fun d(msg: String?) {
        if (!SHOW_LOG) {
            return
        }
        if (msg.isNullOrEmpty()) {
            return
        }
        Log.d(TAG, msg)
    }

    fun i(msg: String?) {
        if (!SHOW_LOG) {
            return
        }
        if (msg.isNullOrEmpty()) {
            return
        }
        Log.i(TAG, msg)
    }

    fun w(msg: String?) {
        if (!SHOW_LOG) {
            return
        }
        if (msg.isNullOrEmpty()) {
            return
        }
        Log.w(TAG, msg)
    }

    fun e(msg: String?) {
        if (!SHOW_LOG) {
            return
        }
        if (msg.isNullOrEmpty()) {
            return
        }
        Log.e(TAG, msg)
    }
}